Gwen writing (#9)
jbae11/depletion_rom: docs/introduction.tex

high and low resolution. 
An example of a high resolution element is the spent fuel 
isotopic composition from a single fuel bundle, and an example 
of a low resolution element is the total fuel utilization in 
the system. 
The intention behind the use of \gls{NFC} simulators is to develop 
a better understanding of the dependence between various components 
in the system and the effects of changes on the system. 
Their goal is to assist in evaluating and improving potential 
strategies for nuclear power development in terms of improving waste 
management, economic competitiveness, etc \cite{yacout_modeling_2005}.   

One of the major functionalities of a \gls{NFC} simulator is its 
ability to transmute nuclear fuel in a reactor based on reactor 
conditions such as burnup, enrichment, etc. 
The transmutation results impact the accuracy of the \gls{UNF} 
composition, and thus the capability of using the \gls{NFC} to 
analyze the impact of a \gls{NFC} on variables such as waste profile.  

Current fuel cycle simulator codes include \Cyclus, ORION, DYMOND 
and VISION. 
Each of these NFC simulators obtain transmutation results using 
different methods. 

> Some of these codes have multiple methods for obtaining 

Reviewer: ```suggestion
Some of these codes have multiple options for obtaining 
```

transmutation results. 
There is essentially two overall types of methods to obtain transmutation 
results: recipe-based method and library-based method. 
The recipe-based method is a low fidelity method, while the library-based method is 

> a high fidelity method. 

Reviewer: I think this sort of binary division into low and high fidelity is not the best option. Maybe just get rid of the sentence?

In the recipe-based method, direct neutronics calculations are not performed 
within the model, it is done externally \cite{yacout_vision_2006}. 
The resulting fresh and spent fuel compositions for specific parameters 
is entered directly into the \gls{NFC} model \cite{sunny_transition_2015}. 
In the library-based method,  the \gls{NFC} code dynamically calculates depleted 
fuel recipes by interpolating reactor data libraries generated by
reactor physics burnup calculation code such as SERPENT 

> \cite{leppanen_serpent_2013}, Origen \cite{croff_users_1980}, etc. 

Reviewer: ```suggestion
\cite{leppanen_serpent_2013}, \cite{ORIGEN} \cite{croff_users_1980}, and etc. 
```

Table \ref{tab:nfc_code} shows a simple breakdown of the 
transmutation methods available for each \gls{NFC} code. 

\begin{table}[h]
    \centering
    \label{tab:nfc_code}
    \begin{tabular}{lrrr}
        \hline
        \gls{NFC} code & Transmutation Methods Available \\
        \hline
        \Cyclus & Recipe-based and Library-based\\
        ORION & Recipe-based and Library-based\\
        DYMOND & Recipe-based  \\
        VISION & Recipe-based  \\
        \hline
    \end{tabular}
    \caption{\gls{NFC} Methods for transmutation in reactor module}
\end{table}

> \Cyclus has three methods to obtain transmutation results: \Cycamore 

Reviewer: ```suggestion
\Cyclus has three methods to obtain transmutation results: the \Cycamore 
```

> recipe reactor \cite{huff_extensions_2014} CyBORG 

Reviewer: ```suggestion
recipe reactor \cite{huff_extensions_2014}, CyBORG 
```

> \cite{skutnik_cyborg_2016} and Bright-lite \cite{flanagan_brightlite_2014}.  

Reviewer: ```suggestion
\cite{skutnik_cyborg_2016}, and Bright-lite \cite{flanagan_brightlite_2014}.  
```

The \Cycamore recipe reactor accepts a fresh and spent fuel recipe that is 
defined by the user. 

> CyBORG calls the Origen solver \gls{API} directly to calculate depleted fuel

Reviewer: ```suggestion
CyBORG calls the \cite{ORIGEN} solver \gls{API} directly to calculate depleted fuel
```

> compositions, which then becomes the output recipe for the CyBORG reactor 

Reviewer: ```suggestion
compositions, which then becomes the output fuel composition.
```

> in \Cyclus \cite{skutnik_cyborg_2016}. 

Reviewer: ```suggestion
```

> Bright-lite uses libraries that represent a specific reactor design to 

Reviewer: this is quite unclear. How does it calculate the burnup? Maybe it helps to illustrate what the user puts in and how the algorithm uses the user inputs, and what the algorithm outputs

calculate the burnup, and output isotopic vectors for a given input fuel
\cite{flanagan_brightlite_2014}. 
ORION has a recipe-based and library-based method to obtain transmutation results. 
In ORION's library-based method, burnup-dependent cross section libraries 
for multiple reactor types with multiple initial fuel enrichment are 
generated before ORION analysis \cite{sunny_transition_2015}. 
The libraries are then used to generate spent fuel recipes based on 

> reactor conditions. 

Reviewer: what conditions? like power?

> DYMOND has a recipe-based method where fuel cycle recipes for both the input 

Reviewer: ```suggestion
DYMOND has a recipe-based method where recipes for both the input 
```

and output fuel compositions are externally calculated. 
Isotopes are tracked in lumped categories: fission products, minor 
actinides, Uranium and Plutonium \cite{feng_standardized_2016}.  
VISION is an extention of the DYMOND code. 

> It has a similar recipe-based method as the DYMOND code, but with isotopic 

Reviewer: ```suggestion
It has a similar recipe-based method as the DYMOND code, but with individual isotopic 
```

tracking \cite{yacout_vision_2006}. 

For \gls{NFC} simulators, striking a balance between fidelity 
and computational cost is a key issue. 
The advantage of using high fidelity models that have a physics-based 

> approach is its inherent flexibility to readily accommodate a broad 

Reviewer: This paragraph is weird.. isn't it? Maybe cut it into many sentences?

space of potential isotopic vectors that are present when modeling 
complex scenarios that involve multi-reuse fuel and isotopic changes 
that occur during a transition or on the approach to an equilibrium 
\cite{sunny_transition_2015}. 
However, using high fidelity models for century-long simulations 
can result in impractical computational times. 
The advantage of using low fidelity models (recipe-based method)
is the low computational cost. 
It is an acceptable method for modeling fuel cycles with fixed input 
composition or fuel cycles at equilibrium \cite{sunny_transition_2015}. 
However, for fuel cycles not at equilibrium, it can result in less 
accurate results. 

Therefore, to find a middle ground of accurate depletion data while 
maintaining a practical computational cost, this paper introduces 

> a reduced order model that is able to predict \gls{PWR} \gls{UNF}

Reviewer: ```suggestion
a trained neural network model that is able to predict \gls{PWR} \gls{UNF}
```
